lib/components/src/ActionBar/ActionBar.stories.tsx

```diff
@@ -11,7 +11,7 @@ export default {
   Component: ActionBar,
   title: 'Basics|ActionBar',
   decorators: [
-    storyFn => (
+    (storyFn: () => JSX.Element) => (
```

Comment:
`JSX` comes from the global, which is subject to change and may not be future proof...  Here is [the reason with some tips](https://medium.com/@martin_hotell/10-typescript-pro-tips-patterns-with-or-without-react-5799488d6680#58e3) that I found super useful.

Reply:
I didn't know about that, good article!